app/lib/stellar/__tests__/networks.test.ts

```diff
@@ -15,7 +15,6 @@ describe('hostnameToNetwork', () => {
     expect(hostnameToNetworkType('futurenet.local')).toEqual(networks.future)
 
     // unknown hosts default to local
-    expect(hostnameToNetworkType()).toEqual(networks.local)
```

Comment:
[note] 

`hostnameToNetworkType` expects one argument.
So it's grammatically incorrect.
So I removed it.